src/propsheet/PropSheetHandler.cpp

```diff
@@ -34,7 +34,7 @@ class ConsolePropertySheetHandler WrlFinal : public RuntimeClass<RuntimeClassFla
     }
 
     // IPersist
-    STDMETHODIMP GetClassID(_Out_ CLSID* clsid) override
+    STDMETHODIMP GetClassID(_Out_ CLSID * clsid) override

```

Comment:
FORMAT: This whole file is filled with the issues I called out in the body.

Reply:
There is some serious Objective-C vibe going around this one...

Reply:
Ah, objective-C, my other favorite language :)